sycl/unittests/pi/cuda/test_interop_get_native.cpp

```diff
@@ -19,17 +19,15 @@ using namespace cl::sycl;
 struct CudaInteropGetNativeTests : public ::testing::TestWithParam<platform> {
 
 protected:
-  queue syclQueue_;
-  context syclContext_;
+  std::unique_ptr<queue> syclQueue_;
   device syclDevice_;
 
   void SetUp() override {
     syclDevice_ = GetParam().get_devices()[0];
-    syclQueue_ = queue{syclDevice_};
-    syclContext_ = syclQueue_.get_context();
+    syclQueue_ = std::unique_ptr<queue>{new queue{syclDevice_}};
   }
 
-  void TearDown() override {}
+  void TearDown() override { syclQueue_.release(); }
```

Comment:
This looks unnecessary to me, this will cause memory leak (we just release ownership, but object will be never destroyed).